src/Reflection/ReflectionParser.php

```diff
@@ -26,7 +27,7 @@ final class ReflectionParser
     public function __construct()
     {
         $parserFactory = new ParserFactory();
-        $this->parser = $parserFactory->create(ParserFactory::PREFER_PHP7);
+        $this->parser = $parserFactory->createForVersion(PhpVersion::fromString('7.4'));
```

Comment:
PHPParser 5 has removed the `create` method. I used the `createForVersion` instead, setting this as 7.4. I am not really sure that this is the right solution

Reply:
I think there is some `createForLatestPhp()`, that should be used instead so we don't create fixed value here

Reply:
`createForNewestSupportedVersion()` should work, there is rule for that

https://github.com/rectorphp/rector-php-parser/blob/9639ef9429f74c14fdc3c7dad9d3b0d916663de0/src/Rector/MethodCall/ParserFactoryRector.php#L24-L48